modules/dublincore/src/main/java/org/opencastproject/metadata/dublincore/DublinCoreGsonAdapter.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package org.opencastproject.metadata.dublincore;
+
+
+import com.google.gson.JsonParser;
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+import org.json.simple.parser.ParseException;
```

Comment:
Well, I am not throwing that exception. I am catching it. Because I'm using an existing method which uses JSON simple. It's a bit stupid but was the easiest solution. Everything else would have required writing lots more code and likely duplicating logic. Kinda annoying to have multiple JSON libraries.